test/rebar3_hex_build_SUITE.erl

```diff
@@ -63,8 +67,18 @@ create_package_with_alias_deps(Config) ->
 create_docs_test(Config) ->
     StubConfig = #{type => app, dir => data_dir(Config), name => "valid"},
     {State, Repo, App} = test_utils:make_stub(StubConfig),
-    {ok, _Docs} = rebar3_hex_build:create_docs(State, Repo, App),
-    ok.
+    {ok, #{tarball := Tarball}} = rebar3_hex_build:create_docs(State, Repo, App),
```

Comment:
Added a test to ensure we don't create docs tarball with files insides a "docs" dir.